bin/dotd

```diff
@@ -236,6 +237,26 @@ def print_dtp_message
   end
 end
 
+# Retrieves recent Honeybadger issues, displaying them on the console.
+def get_honeybadger_issues
+  issues = Honeybadger.get_recent_issues
+
+  if issues.count > 0
+    puts ''
+  end
+  issues.each do |issue|
+    message = "(#{issue[:url]} [#{issue[:environment]}, #{issue[:project]}]) #{issue[:message]}"
```

Comment:
Could even bold the environment/project block to help the eye pick out the start of each issue.